cfme/services/myservice.py

```diff
@@ -132,12 +211,12 @@ def retire(self):
             assert vm_state in ['off', 'unknown']
 
     def retire_on_date(self, retirement_date):
-        navigate_to(self, 'SetRetirement')
-        fill(retirement_form, {'retirement_date': retirement_date},
-             action=form_buttons.save)
-        navigate_to(self, 'Details')
+        view = navigate_to(self, 'SetRetirement')
+        view.retirement_date.fill(retirement_date)
```

Comment:
@mfalesni : Getting error here  http://pastebin.test.redhat.com/470984. 
If I change "if self.browser.get_attribute("ng-model", self) is not None:" to "if self.browser.get_attribute("ng-model", self) is None: . The calendar field is filled .